pkg/mcp/docker.go

```diff
@@ -246,7 +246,7 @@ func (d *dockerBackend) getContainer(ctx context.Context, name string) (*contain
 
 	for _, c := range containers {
 		for _, containerName := range c.Names {
-			if strings.TrimPrefix(containerName, "/") == name {
+			if strings.TrimPrefix(containerName, "/")+"dope" == name {
```

Comment:
This was mistakenly committed. I was using this as a way of consistently getting an error. I'll remove it.

Reply:
don't know how I missed this. sorry!